cpp/doxygen/developer_guide/DEVELOPER_GUIDE.md

```diff
@@ -152,15 +152,17 @@ The following guidelines apply to organizing `#include` lines.
    from other RAPIDS libraries, then includes from related libraries, like `<thrust/...>`, then
    includes from dependencies installed with cuDF, and then standard headers (for example
    `<string>`, `<iostream>`).
- * Use `<>` instead of `""` unless the header is in the same directory as the source file.
+ * We use clang-format for grouping and sorting headers automatically. See the
+   `cudf/cpp/.clang-format` file for specifics.
+ * Use `<>` for all includes except for internal headers that are not in the `include/cudf`
+   directory. In other words, if it is a cuDF internal header (e.g. in the `src` or `test`
```

Comment:
Does this apply to `cudf_test`? That means we will need to change all test files to use `#include "cudf_test/..."`.

Reply:
No, because `cudf_test` is a library with public headers in `include/cudf_test`. Therefore includes of its *public* headers should use `<>`. Moreover, there is a separate regex grouping for `cudf_test` in .clang-format.

Reply:
Added clarification of this (and nvtext) in the guide.